package errorHandler

func Catch(err error) error {
	return err
}
